NitroxServer-Subnautica/GameLogic/Entities/Spawning/ReefbackBootstrapper.cs

```diff
@@ -10,6 +11,7 @@ namespace NitroxServer_Subnautica.GameLogic.Entities.Spawning.EntityBootstrapper
     class ReefbackBootstrapper : IEntityBootstrapper
     {
         private float creatureProbabiltySum = 0;
+        private readonly bool aprilFools = DateTime.Today.Month == new DateTime(1, 4, 1).Month && DateTime.Today.Day == new DateTime(1, 4, 1).Day;
```

Comment:
I think this can be simplified with something like that
```
DateTime.Now.Month == 4 && DateTime.Now.Day == 1
```

Reply:
Or `DateTime.Today == new DateTime(DateTime.Today.Year, 4, 1)`

Reply:
`DateTime.Today is {} today && today.Month == 4 && today.Day == 1`